server/signal.go

```diff
@@ -49,7 +49,7 @@ func (s *Server) handleSignals() {
 				s.Debugf("Trapped %q signal", sig)
 				switch sig {
 				case syscall.SIGINT:
-					s.Noticef("Server Exiting..")
+					s.Shutdown()
```

Comment:
I would do an s.Notice("Initiating Shutdown...")
then after Shutdown() do the original Server Exiting.

Reply:
s.Shutdown() is already logging s.Noticef("Server Exiting.."). If I do what you ask will need to verify that some tests are not looking "Server Exiting" in logging.. will let you know.

Reply:
Also note that if we had a proper logger that say would be file based and closed on Shutdown() then tracing after that would "fail".

Reply:
oh so we were printing it twice?

Reply:
nvm, understand now.

Reply:
No, remember we were not shutting down :-)
I have moved the initiating shutdown in s.Shutdown() and will print exiting before os.Exit(). Running tests and checking on Windows, will let you know.

Reply:
ok

Reply:
Took this approach which I think is best:
- Moved "Initiating Shutdown..." at beginning of Shutdown() function
- At the end of this function, before closing the logger, log "Server Exiting.."
Since Shutdown() is invoked prior to os.Exit() it will be equivalent of having the logging just before os.Exit() but with the correct behavior of logging before closing the logger. It also avoid having to put the tracing in 3 different places :-)

Reply:
Sounds good.